### PR TITLE
[103] Semantics labels for icons

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: mdc_100_series/lib/home.dart [hunk 1]
```diff
@@ -85,21 +85,30 @@ class HomePage extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         leading: IconButton(
-          icon: Icon(Icons.menu),
+          icon: Icon(
+            Icons.menu,
+            semanticLabel: "menu",
+          ),
```
Comment (jonahwilliams) [inline mdc_100_series/lib/home.dart]: Ultra nit: use `'` instead of `"` to be consistent

File: mdc_100_series/lib/home.dart [hunk 2]
```diff
@@ -85,21 +85,30 @@ class HomePage extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         leading: IconButton(
```
Comment (jonahwilliams) [inline mdc_100_series/lib/home.dart]: Not sure how important this is for the codelab, but all of the `Icon` constructors could be made `const`:

File: mdc_100_series/lib/home.dart (hunk 1 above)
Comment (willlarche) [reply mdc_100_series/lib/home.dart]: Done.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**